JOSESwift/Sources/SecureEnclaveSigner.swift

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
This should probably be the Apache 2 license

Reply:
Yea, I was unsure about what to put here.

Reply:
You can find the Apache one in one of the other open source plug-ins/projects.